lib_test/mux_test.ml

```diff
@@ -6,14 +6,14 @@ module TestPacket = struct
 
   type request_hdr = {
     req_id : int;
-    req_payload : string;
+    req_payload : bytes;
   } [@@deriving sexp]
 
   type request_body = bytes
 
   type response_hdr = {
     res_id : int option;
-    res_payload : string;
+    res_payload : bytes;
```

Comment:
Why does this need to be bytes?

Reply:
Because we were using it as bytes at some places.
Converting it to bytes seemed to be the easier solution.

Reply:
Are we actually modifying it though? I think bytes is good as a temporary buffer in some local function, but we should prefer to use strings wherever we can, otherwise we end up with a lot more bytes<->string conversion than necessary.

Reply:
I've started that way, but realized that keeping it string would require much more and messier changes, so I've decide to keep things simple, since this is just test code.

Reply:
Good point, I haven't noticed this is lib_test.